store/sql_user_store.go

```diff
@@ -437,13 +439,13 @@ func (s SqlUserStore) GetEtagForDirectProfiles(userId string) StoreChannel {
 	return storeChannel
 }
 
-func (s SqlUserStore) GetEtagForAllProfiles() StoreChannel {
+func (s SqlUserStore) GetEtagForAllProfiles(limit int, offset int) StoreChannel {
```

Comment:
Yeah, this is something I wanted to put more thought into. I think it's fine to just remove it and rely on the client being smarter about what it's requesting. If it ever starts to be slow then we can look at adding it back in